### ISSUE TITLE
Exclude content representation

### ISSUE THREAD

**Issue by nilshoerrmann** [afbora/kirby-minify-html#3]
We have an RSS feed on our side that we provide via a content representation: with this extension, the XML output is also minified which results in invalid markup.

Would it be possible to check the content type and only minify output that is explicitely HTML?

**Comment by nilshoerrmann** [comment]
This also becomes an issue when sending plain text email, where all whitespace gets removed. What about checking the content type before applying the minification?

```php
class MinifyHTML extends Template
{
    public function render(array $data = []): string
    {
        $html = Tpl::load($this->file(), $data);

        if (option('afbora.kirby-minify-html.enabled') === true && $this->type() === 'html') {
            // ...

            return $htmlMin->minify($html);
        }

        return $html;
    }
}
```

**Comment by afbora** [comment]
Thank you for digging @nilshoerrmann  Have you tried mentioned solution and it works?

**Comment by nilshoerrmann** [comment]
Yes, it worked for my email issue. But I don‘t know enough about how Kirby handles the type to understand if there are any undesired side effects. If it‘s save to assume that the type always returns either HTML or the type of a content representation, this fix should be fine.

**Comment by afbora** [comment]
I've released v1.5.0 with this fix.
Thanks @nilshoerrmann
